classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
```

Comment:
- [ ] OPTIONAL: Since this is an array and is supposed to hold an array of students, I strongly suggest to make the variable plural to avoid confusion 👍🏻